docs/troubleshooting/detections/detection-rules.asciidoc

```diff
@@ -76,3 +76,40 @@ You can resolve this by expanding the time range, or by configuring {kib}'s auto
 
 CAUTION: Turning off `autocomplete:useTimeRange` could cause performance issues if the data set is especially large.
 ====
+
+[discrete]
+[[rule-exceptions-field-conflicts]]
+.Warning about type conflicts and unmapped fields 
+[%collapsible]
+====
+
+A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields that have <<fields-with-conflicting-types,type conflicts>> or are <<unmapped-field-conflict,unmapped>> across multiple indices. You can learn more about the conflict by hovering over the field. After you select it, the warning message is displayed beneath the field.
+
+NOTE: A field can have type conflicts _and_ be unmapped in specified indices.  
+
+[role="screenshot"]
+image::images/warning-icon-message.png[Shows the warning icon and message,70%]
+
+[float]
+[[fields-with-conflicting-types]]
+==== Fields with conflicting types  
+
+Type conflicts occur when a field is mapped to different types across multiple indices. To resolve this issue, consider creating new indices with the correct type mappings and {ref}/docs-reindex.html[reindexing your data]. Otherwise, use the information about a field's type mapping to ensure you're entering correct field values when defining exception conditions. 
```

Comment:
```suggestion
Type conflicts occur when a field is mapped to different types across multiple indices. To resolve this issue, you can create new indices with matching field type mappings and {ref}/docs-reindex.html[reindex your data]. Otherwise, use the information about a field's type mappings to ensure you're entering compatible field values when defining exception conditions. 
```